<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="studentInfoEditor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Student Info Editor</string>
    </property>
    <widget class="QLabel" name="idNumberLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>ID Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="yearLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Year Level</string>
     </property>
    </widget>
    <widget class="QLabel" name="coursePick">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Course</string>
     </property>
    </widget>
    <widget class="QLabel" name="genderLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idNumberInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="coursePicker">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="yearLevelPicker">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="genderPicker">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="idNumberInput_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="hypen">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;-&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="courseInfoEditor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Course Info Editor</string>
    </property>
    <widget class="QLabel" name="courseCodelabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Course Code</string>
     </property>
    </widget>
    <widget class="QLabel" name="courseNamelabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Course Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="courseCodeInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="courseNameInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addStudent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(5, 132, 3);
color: white;</string>
    </property>
    <property name="text">
     <string>Add Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editStudent">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(182, 182, 182);
color: white;</string>
    </property>
    <property name="text">
     <string>Edit Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteStudent">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(207, 0, 0);
color: white;
</string>
    </property>
    <property name="text">
     <string>Delete Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCourse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(5, 132, 3);
color: white;</string>
    </property>
    <property name="text">
     <string>Add Course</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteCourse">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(207, 0, 0);
color: white;
</string>
    </property>
    <property name="text">
     <string>Delete Course</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="studentInfoDisplay">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>601</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>ID_Number</string>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Year Level</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gender</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course Code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Enrollment Status</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="courseListDisplay">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>40</y>
      <width>271</width>
      <height>601</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Course Code</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course Name</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="studentLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search Student:</string>
    </property>
   </widget>
   <widget class="QLabel" name="courseLabel">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search Course:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createCSV">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
font-size: 15px;</string>
    </property>
    <property name="text">
     <string>Create File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchStudent">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchCourse">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="searchbyLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search By:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="searchFilter">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="editCourse">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(182, 182, 182);
color: white;</string>
    </property>
    <property name="text">
     <string>Edit Course</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
